inventory.py

```diff
@@ -49,6 +58,7 @@ def add_product():
         if input(prompt).upper() == 'N':
             add_more = False
 
+    #once the user has added all of the products, write products to file.
     if os.path.exists(file_name):
```

Comment:
you don't need to delete the file in order to write to it if you open the file with the correct mode (the mode is `w`, if I remember correctly.)

Reply:
I have that in mind too but I'm planning later to sort the products and the inventory after the user has entered new records. I intend to do the sorting on the dictionary and then re-write all records to the csv file. I thought it's more efficient to just delete the file and then create an empty one to write to rather than deleting all the contents to re-write the whole thing.

Reply:
But I will look into that again once I'm more or less done with most of the functions. Thanks.